src/retrace/retrace.py

```diff
@@ -2032,6 +2031,10 @@ def convert_flattened_format(self) -> bool:
             log_error("Failed to convert flattened vmcore %s - errno(%d - '%s')" %
                       (self._vmcore_path, e.errno, e.strerror))
             return False
+        except TimeoutExpired:
+            log_error("WARNING: command: '{}' exceeded {} "
```

Comment:
Is this a warning or an error? Since the process fails overall due to the timeout, I'd call it an error in the message as well.

Reply:
Hmmm, yeah it's probably an error so I'll remove / change that "WARNING"